src/App.tsx

```diff
@@ -1,20 +1,35 @@
-import Logo from "@assets/logo.png";
-import DataContainer from "@components/DataContainer/DataContainer";
-import HelloWorld from "@components/HelloWorld/HelloWorld";
+// import Logo from "@assets/logo.png";
+// import DataContainer from "@components/DataContainer/DataContainer";
+// import HelloWorld from "@components/HelloWorld/HelloWorld";
 
-import styles from "./App.module.css";
+// import styles from "./App.module.css";
 
-export default function App() {
+// export default function App() {
+//   return (
+//     <main className={styles.main}>
+//       <img className={styles.logo} alt="React logo" width="400px" src={Logo} />
+//       <DataContainer />
+//       <div>
+//         {`VITE_SOME_ENVIRONMENT_VARIABLE from .env.local: "${
+//           import.meta.env.VITE_SOME_ENVIRONMENT_VARIABLE
+//         }"`}
+//       </div>
+//       <HelloWorld msg="Hello EE Practicum Team 2!" />
+//     </main>
+//   );
+// }
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from "react";
+import ActivityList from "./ActivityList.js";
```

Comment:
I know you said the import isn't working correctly. After you move the `ActivityList` to the new folder inside `/components`, please try following the same format as the examples on lines 2-3. 

We're using the `@` shortcuts to make it easier to import from important folders without ever having to use `../../../` to go up to parent folders. 

You can also see here that the import for this project expects the file extensions to be left off entirely:

```suggestion
import ActivityList, { Activity } from "@components/ActivityList/ActivityList";
```